harness/determined/cli/master.py

```diff
@@ -70,8 +71,37 @@ def logs(args: argparse.Namespace) -> None:
         print(format_log_entry(response.logEntry))
 
 
+def set_message(args: argparse.Namespace) -> None:
+    sess = cli.setup_session(args)
+
+    if args.message is None:
+        raise ValueError("Provide a message using the -m flag.")
+    body = bindings.v1SetClusterMessageRequest(
+        startTime=args.start, endTime=args.end, message=args.message, duration=args.duration
```

Comment:
probably worth checking these user-inputted timestamps, since we expect them in a specific format, and throwing an exception if it's invalid.

we have `is_protobuf_timestamp` in `determined.common` for this purpose